plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -44,13 +44,14 @@ export class GithubPublisher implements PublisherBase {
     values: RequiredTemplateValues & Record<string, JsonValue>,
   ) {
     const [owner, name] = values.storePath.split('/');
+    const description = String(values.description);
```

Comment:
I'm a typeed language beginner so I would appreciate some guidance on what to do here. `description` is a `JsonValue` but `createInOrg` expects it to be `undefined | string`.

Surely there is a better solution than casting it like this? I mean, if I was sending 20 different to GitHub from the `values` would I have to cast all of them?

Reply:
You should be able to do `values.description as string`

Reply:
Or in the `createInOrg` call you can do `description: description ?? ''` but the first one is better.

Reply:
Thanks Ben. That looks more like Typescript to me.